docs/property_names.js

```diff
@@ -8,7 +8,8 @@ export {
 };
 
 const damageTag = 'DAMAGE PERCENTAGE';
-const geoidTag = 'GEOID';
+// Unique name to avoid clashing with user-given columns.
```

Comment:
Can you add a note that this is never actually exposed to the end user

Reply:
Done.